### DISCUSSION TITLE
MFEM 3.4 Released

### DISCUSSION THREAD

**Discussion by tzanio** [mfem/mfem#3483] [Team Posts]
Dear MFEM community,

[Version 3.4](https://github.com/mfem/mfem/blob/v3.4/CHANGELOG) of MFEM was released on May 29, 2018. Some of the new additions in this release are:

- Significantly improved non-conforming unstructured AMR scalability.
- Integration with PUMI, the [Parallel Unstructured Mesh Infrastructure](https://scorec.rpi.edu/pumi/) from RPI.
- Block nonlinear  operators and variable order NURBS.
- Conduit [Mesh Blueprint](http://llnl-conduit.readthedocs.io/en/latest/blueprint.html) support.
- General "high-order"-to-"low-order refined" field transfer.
- New specialized time integrators (symplectic, generalized-alpha).
- Twelve new examples and miniapps.

To download, and for more information visit http://mfem.org.

We welcome any and all feedback at https://github.com/mfem/mfem/issues.

**Comment by IanLee1521** [comment]
See also: https://software.llnl.gov/news/2018/05/29/mfem-3.4/
